src/pages/UploadPage.tsx

```diff
@@ -1,10 +1,96 @@
-import React from "react";
+import * as React from "react";
 import Container from "@mui/material/Container";
+import Button from "@mui/material/Button";
+import Box from "@mui/material/Box";
+import TextField from "@mui/material/TextField";
+import API from "../api/api";
 
 export default function UploadPage() {
+  const [cover_letter, setCoverLetter] = React.useState<File | null>(null);
+  const [resume, setResume] = React.useState<File | null>(null);
+
+  const sendCoverLetter = () => {
+    if (!cover_letter) {
+      return;
+    } else {
+      API.getAPI().addCoverLetter(cover_letter);
+    }
+  };
+  const sendResume = () => {
+    if (!resume) {
+      return;
+    } else {
+      API.getAPI().addResume(resume);
+    }
+  };
+
   return (
     <Container maxWidth="lg" sx={{ mt: 10, mb: 4 }}>
-      <h1>UploadPage</h1>
+      <h1>File Upload</h1>
+      <h4>Cover Letter</h4>
+      <Box sx={{ display: "flex", mb: 2, flexDirection: "row" }}>
+        <TextField
+          disabled
+          id="outlined-disabled"
+          label="Cover Letter"
+          defaultValue="Insert File here"
+          size="small"
+          sx={{ width: "300px", marginRight: 2 }}
+          value={cover_letter?.name}
+        />
+        <Button
```

Comment:
Please delete one upload-button and only place one, maybe below both input fields.  For setting the state of both inputs maybe add another button per input or, if possible, set the state by clicking inside the input field.